nix/shell/spdk.nix

```
@@ -18,6 +18,15 @@ let
     echo "FIO path        : $FIO"
   '';
 
+  # Determine the system architecture
+  system = builtins.currentSystem;
+
+  # Define CFLAGS based on the architecture
+  cflagsValue =
+    if system == "aarch64-linux"
+    then "-march=armv8-a"
```

to match what the script does?
```suggestion
    then "-march=armv8-a+crypto"
```

---

Good call.

---

are we compiling spdk with crypto by default for arm?

---

In the `build_spdk.sh` script we have spdk on arm compiling with crypto implicitly I believe based on this snippet:
https://github.com/openebs/spdk-rs/blob/93d0d206e48961f02b54bf32d87b13726027974c/build_scripts/build_spdk.sh#L261-L271
This could always be amended for synchronicity.

---

Yeah that's where I had seen it

---

This is now fixed.